zerver/lib/outgoing_webhook.py

```diff
@@ -190,6 +191,9 @@ def notify_bot_owner(event: Dict[str, Any],
         notification_message += "\nWhen trying to send a request to the webhook service, an exception " \
                                 "of type %s occurred:\n```\n%s\n```" % (
                                     type(exception).__name__, str(exception))
+    if failure_message:
+        notification_message += "\nThe error occurred during request to the webhook service:\n" \
```

Comment:
This error message isn't going to make a lot of sense to users.  Let's change this to:

"Your outgoing webhook bot %(bot_email)s received an error when posting a message:"